lib/ex_aws/s3/client.ex

```diff
@@ -312,15 +313,24 @@ defmodule ExAws.S3.Client do
   defcallback put_object_acl!(bucket :: binary, object :: binary, acl :: acl_opts) :: ExAws.Request.success_content
 
   @type pub_object_copy_opts :: [
-    {:metadata_directive, :copy | :replace}
+    {:metadata_directive, :COPY | :REPLACE}
```

Comment:
aside from `:DOWN` and a few others `:FOO` style atoms aren't super common. Leaving them downcase means I need to capitalize them in ex_aws I assume yes?